.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_LOCALURL=http://localhost:3000/api
+REACT_APP_HEROKUURL=https://newsroom-april2020-api.herokuapp.com/api
```

Comment:
I think this is supposed to be .gitignored..! Ali told me.

Reply:
Yeah, but not so much importance at the moment, there is no API KEY

Reply:
Okay, but make sure to ignore this later...